flathub.json

```diff
@@ -0,0 +1,6 @@
+{
+    "only-arches": [
+        "x86_64"
```

Comment:
:-(

Reply:
Sorry about this. 
Cross-build from Linux x64 host to Linux arm64 target is not currently supported, I have some difficulty to build and test the aarch version, I think I will add aarch support in the future.